### PR TITLE
Adjusting instance type priorities.

### PR DESCRIPTION
**Pull request by ca16**
Have p2.xlarge have higher priority than p2.8xlarge. This means ASGs with p2.xlarge instances will get scaled up ahead of ASGs with p2.8xlarge instances (assuming both instance types have the resources to support the pending pod).

Also removed a comment that no longer applies.

### CODE REVIEW

File: autoscaler/cluster.py [hunk 1]
```diff
@@ -54,11 +54,8 @@ class Cluster(object):
     # just prioritize the ones that we're confident about
     _GROUP_DEFAULT_PRIORITY = 10
     _GROUP_PRIORITIES = {
```
Comment (jkinkead) [inline autoscaler/cluster.py:56]: Weird that this isn't configurable.

**Review by jkinkead — APPROVED** [review]
LGTM!

File: autoscaler/cluster.py (hunk 1 above)
Comment (ca16) [reply autoscaler/cluster.py:56]: Right! Could be a nice improvement to add at some point though...

Comment (jkinkead) [reply autoscaler/cluster.py:56]: The flag type I added would make it easy to add this as a commandline flag . . .

;)

Comment (ca16) [reply autoscaler/cluster.py:56]: Exactly what I was thinking too...

**Pull request merged**